### ISSUE TITLE
Replace quantile with built-in function percentile_cont()?

### ISSUE THREAD

**Issue by stenerikbjorling** [biolab/orange3#5432]
- [ ] **What's your use case?**

it is now difficult to do a regular make install of the quantile extension to Postgres for the latest versions of Postgres. In my communication with the developer of PostgresApp he proposed the alternative to use percentile_cont() in the calls to the Postgres environment. Think it would be a good option for future versions of the Orange3 environment and would simplify the installation and full use of the Postgres connectivity.

- [ ] **What's your proposed solution?**

Described above....

- [ ] **Are there any alternative solutions?**

Described above.

**Comment by janezd** [comment]
You're right. The use of this extension was rather an experiment within some project and it should better be replaced by `percentile_cont`.

Could you prepare a pull request, with tests if possible?

**Comment by stenerikbjorling** [comment]
Hi,

I am sorry - I am not a programmer in that sense, cannot pull and compile myself. But maybe this could be a functionality in later versions?

Take care, all the best…

Stene

______

> 14 maj 2021 kl. 12:26 skrev Janez Demšar ***@***.***>:
> 
> You're right. The use of this extension was rather an experiment within some project and it should better be replaced by percentile_cont.
> 
> Could you prepare a pull request, with tests if possible?
> 



Sten-Erik Björling
Enviro Data
Kyrkogatan 5A 2 tr
SE-972 32  Luleå
Sweden

E-Mail: ***@***.***
Mobile: +46-70-655 11 72
Wire: @stenerikbjorling
Skype: stenerikbjorling
iChat: ***@***.***
FaceTime: ***@***.***
whereby.com: stenerikbjorling
Hotmail / Messenger: ***@***.***
GMail: ***@***.***
______

This email and any files transmitted with it are confidential, may be legally privileged and are intended solely for the use of the individual or entity to whom they are addressed. If you are not the intended recipient, please note that any use, distribution, or reproduction of the contents of this email is strictly prohibited and may be unlawful. If you are not the intended recipient, please notify the sender by return email and destroy all copies of the original message including any attachments thereto. Thank you.

Please note that we take reasonable precautions to prevent the transmission of viruses; however, we cannot guarantee that this email or its attachments are free from viruses. We only send and receive emails on the basis that we are not be liable for any loss or damage resulting from the opening of this message and/or attachments.
